src/manifest.json

```diff
@@ -24,9 +24,11 @@
 
   "background": {
     "scripts": [
+      "core/background/hooks.js",
```

Comment:
Not the most elegant way to handle this but works for this case since it's registered first.